routes/additem.js

```diff
@@ -1,9 +1,98 @@
 var express = require('express');
 var router = express.Router();
+var bodyParser = require('body-parser');
+
+// middleware for bodyParser
+router.use(bodyParser.json());
+router.use(bodyParser.urlencoded({extended: true}));
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
   res.render('addItem', { title: 'addItem' });
 });
 
+router.post('/', function(req, res) {
+  var itemName = req.body.itemName;
+  var itemPicture = req.body.itemPicture;
+  var itemPrice = req.body.itemPrice;
+  var itemType = req.body.itemType;
+  var itemDescription = req.body.itemDescription;
+
+  console.log(itemName);
+  console.log(itemPicture);
+  console.log(itemPrice);
+  console.log(itemType);
+  console.log(itemDescription);
+
+  if (itemName.length == 0 || itemPrice.length == 0 || itemType.length == 0 || itemDescription.length == 0) {
```

Comment:
Instead of doing these, try this https://www.w3schools.com/tags/att_input_required.asp in the html - looks much nicer :)

Reply:
But how do you tell the user that you are missing the input from this field?

Reply:
the flash message